src/com/facebook/buck/swift/SwiftLibraryDescription.java

```diff
@@ -371,6 +439,9 @@ public static boolean isSwiftTarget(BuildTarget buildTarget) {
     public Optional<SourcePath> bridgingHeader;
     public ImmutableSortedSet<BuildTarget> deps = ImmutableSortedSet.of();
     public Optional<NativeLinkable.Linkage> preferredLinkage;
+
+    public ImmutableMap<Path, SourcePath> headers = ImmutableMap.of();
```

Comment:
not sure about the reason we need this, can we just passing the headers around without relying on the args. The args is for different purpose.